setup.py

```diff
@@ -1,8 +1,27 @@
+version = '0.1.5'
```

Comment:
Move version var after the imports? Per [PEP8](https://www.python.org/dev/peps/pep-0008/):

> Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.

Reply:
fixed